bin/cli.js

```diff
@@ -48,26 +48,30 @@ const cliFlags = yargs
   .option('config', {
     'describe': 'Path to config file',
     'type': 'string',
+    //equivalent to: cliFlags.config = getConfigFromFile(cliFlags.config);
+    'coerce': getConfigFromFile 
```

Comment:
I though we agreed to leave it as string (path to file) instead of mixing up path and config value...
Am I wrong?